### PR TITLE
Configure jsx-a11y plugin for eslint. 

### PR DESCRIPTION
**Pull request by johnworth**
This also adds a lint script.

I think this is looking over now and commenting on even though it's a WIP. We'll have to fix up the existing components before merging, though, since they have existing a11y issues that are causing the CI actions to fail with the new eslint config.

### CODE REVIEW

**Comment by psarando** [comment]
Yeah, let's give it a shot. Maybe all the warnings should be fixed and included in this PR (as you mentioned), then we can an idea of everything this is catching.

**Comment by johnworth** [comment]
> Yeah, let's give it a shot. Maybe all the warnings should be fixed and included in this PR (as you mentioned), then we can an idea of everything this is catching.

I've updated the pull request to fix the reported a11y issues. It's no longer in the WIP state.

File: src/__tests__/__snapshots__/apps.js.snap [hunk 1]
```diff
@@ -1162,11 +1176,15 @@ exports[`App Tile renders 1`] = `
       <div
         className="_class-avatar-106"
       >
-        <div>
+        <div
+          onClick={[Function]}
+          onKeyPress={[Function]}
```
Comment (ianmcorvidae) [inline src/__tests__/__snapshots__/apps.js.snap]: I hate to be the bearer of bad news, but I think `onKeyPress` is the non-recommended one here D:

https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onkeypress mentions it's deprecated and recommends using onkeydown instead.

Comment (slr71) [reply src/__tests__/__snapshots__/apps.js.snap]: Ah, good catch.

**Pull request merged**